v20.2/spatial-data.md

```diff
@@ -0,0 +1,309 @@
+  POINT (-97 35.369999999999997)
+  POINT (-97.680000000000007 35.780000000000001)
+  POINT (-97.950000000000003 35.93)
+  POINT (-98.170000000000002 35.850000000000001)
+  POINT (-97.680000000000007 35.880000000000003)
+  POINT (-97.879999999999995 36.020000000000003)
+  POINT (-97.950000000000003 36.020000000000003)
+  POINT (-98 35.5)
+  POINT (-97.879999999999995 36.100000000000001)
+  POINT (-97.969999999999999 35.549999999999997)
+  POINT (-96.799999999999997 35.649999999999999)
+  POINT (-97.650000000000006 36.119999999999997)
+  POINT (-98.25 36.299999999999997)
+  POINT (-97.719999999999999 35.780000000000001)
+  POINT (-97.780000000000001 35.850000000000001)
+  POINT (-97.599999999999994 35.920000000000002)
+  POINT (-97.420000000000002 36.030000000000001)
+  POINT (-96.129999999999995 35.979999999999997)
+  POINT (-96.069999999999993 36.020000000000003)
+  POINT (-95.650000000000006 35.630000000000003)
+  POINT (-95.180000000000007 35.950000000000003)
+  POINT (-94.730000000000004 36)
+  POINT (-97.400000000000006 35.32)
+  POINT (-96.400000000000006 36.469999999999999)
+  POINT (-95.579999999999998 34.579999999999998)
+  POINT (-95.219999999999999 34.880000000000003)
+  POINT (-95 35.130000000000003)
+  POINT (-94.780000000000001 35.299999999999997)
+  POINT (-94.700000000000003 35.43)
+  POINT (-94.549999999999997 35.57)
+(69 rows)
+~~~
+
+We can see that almost half of all of the tornadoes in this outbreak began in Oklahoma.
+
+It might be interesting to draw these points on a map.  The image below shows the points from the query above drawn as a simple polygon on a map of Oklahoma. The boxes around the polygon show the spatial index coverings for the polygon.
+
+<img width="100%" src="{{ 'images/v20.2/geospatial/1999-oklahoma-tornado-outbreak-map.png' | relative_url }}" alt="1999 Oklahoma tornado outbreak map view">
+
+(Map data &copy; 2020 Google)
+
+#### Example 2. NYC data from the PostGIS tutorial
+
+Follow the steps below to load the SQL for the NYC data used in the [Introduction to PostGIS](https://postgis.net/workshops/postgis-intro/) tutorial.  CockroachDB can work with the tutorial up to Chapter 20, with the following exceptions:
+
+- Do not try to load Shapefiles via the GUI as shown in the tutorial.  Instead, follow the steps below to load the SQL data directly into CockroachDB.  (We have already converted the tutorial Shapefiles to SQL for you.)
+- We do not support GML or KML data.
+- We do not support SVG.
+
+##### Step 1. Load the NYC data
+
+Clone the data set and change to the directory where the SQL files are:
+
+{% include copy-clipboard.html %}
+~~~ shell
+git clone https://github.com/otan-cockroach/otan-scripts/
+~~~
+
+{% include copy-clipboard.html %}
+~~~ shell
+cd otan-scripts/geospatial/geospatial_sql
+~~~
+
+Load the SQL files into your CockroachDB cluster:
+
+{% include copy-clipboard.html %}
+~~~ shell
+cat *sql |cockroach sql --insecure --host=localhost --port=26257
```

Comment:
`cat otan-scripts/geospatial/geospatial_sql/*.sql | cockroach sql --insecure --host=localhost --port=26257`

should do it.